docs/_embed/partner_api.md

```diff
@@ -675,7 +675,7 @@ You'll use the `create_url` (available in a [Zap template object](/partner_api/e
 
 You can prefill the name of a Trello card (field: `name`) in the second step of the Zap template:
 
-`https://zapier.com/webintent/create-zap/?template=113&steps[1][params][name]=hello`
+`https://api.zapier.com/webintent/create-zap/?template=113&steps[1][params][name]=hello`
```

Comment:
I think we want it to be `https://api.zapier.com/v1/embed/...` (the `create_url` from the v1/zap-template payload)

Reply:
Sorry for coming out of nowhere, but I _guess_ it's https://zapier.com to render the template?

![](https://cdn.zappy.app/e9848d1978e0c267f7b5cd40e94d858b.png)

. (There are a few URLs pertaining to this [section](https://platform.zapier.com/embed/partner-api#prefill-options) right below. They should have the same (sub)domain name.)